subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -21,20 +21,42 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #endif
 
 #define SERVER_VERSION_MAJOR 1
+#if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
+#define SERVER_VERSION_MINOR 1
+#define SERVER_MAX_ID		 24
+#else
 #define SERVER_VERSION_MINOR 0
+#define SERVER_MAX_ID		 9
+#endif /* defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1) */
 
 /* Server resource IDs */
-#define SERVER_SHORT_SERVER_ID		0
-#define SERVER_LIFETIME_ID		1
-#define SERVER_DEFAULT_MIN_PERIOD_ID	2
-#define SERVER_DEFAULT_MAX_PERIOD_ID	3
-#define SERVER_DISABLE_ID		4
-#define SERVER_DISABLE_TIMEOUT_ID	5
-#define SERVER_STORE_NOTIFY_ID		6
-#define SERVER_TRANSPORT_BINDING_ID	7
-#define SERVER_REG_UPDATE_TRIGGER_ID	8
-
-#define SERVER_MAX_ID			9
+#define SERVER_SHORT_SERVER_ID							0
```

Comment:
Is your editor configured right? Note Zephyr uses 8 spaces/tab.

Reply:
Let me check that. Do you want numbers to same line or just add tab?